documentation/sphinx/source/release-notes.rst

```diff
@@ -7,21 +7,23 @@ Release Notes
 
 Features
 --------
+
 * Improved team collection for data distribution that builds a balanced number of teams per server and gurantees that each server has at least one team. `(PR #1785) <https://github.com/apple/foundationdb/pull/1785>`_.
 * Added the option to have data distribution FetchKeys to run at a lower priority by setting the knob ``FETCH_KEYS_LOWER_PRIORITY`` `(PR #1791) <https://github.com/apple/foundationdb/pull/1791>`_.
-
 * CMake is now our official build system. The Makefile based build system is deprecated.
-
 * Added local ratekeeper, to throttle reads at a per-storage-process level. `(PR #1447) <https://github.com/apple/foundationdb/pull/1477>`_.
-
 * FDB backups based on disk snapshots, provides an ability to take cluster level backup based on disk level snapshots of storage, tlogs and coordinators. `(PR #1733) <https://github.com/apple/foundationdb/pull/1733>`_.
+* Improved the speed of recoveries on large clusters. `(PR #1729) <https://github.com/apple/foundationdb/pull/1729>`_.
 
 Performance
 -----------
 
 * Use CRC32 checksum for SQLite pages. `(PR #1582) <https://github.com/apple/foundationdb/pull/1582>`_.
 * Added a 96-byte fast allocator, so storage queue nodes use less memory. `(PR #1336) <https://github.com/apple/foundationdb/pull/1336>`_.
 * Handle large packets better. `(PR #1684) <https://github.com/apple/foundationdb/pull/1684>`_.
+* A new Transaction Log spilling implementation is now the default.  Write bandwidth and latency will no longer degrade during storage server or remote region failures.  `(PR #1731) <https://github.com/apple/foundationdb/pull/1731>`_.
```

Comment:
I don't know if this has any effect on how the documentation will be rendered, but just in case:

```suggestion
* A new Transaction Log spilling implementation is now the default.  Write bandwidth and latency will no longer degrade during storage server or remote region failures. `(PR #1731) <https://github.com/apple/foundationdb/pull/1731>`_.
```